triggers/service/src/main/scala/com/digitalasset/daml/lf/engine/trigger/ServiceConfig.scala

```diff
@@ -24,6 +24,7 @@ case class ServiceConfig(
     timeProviderType: TimeProviderType,
     commandTtl: Duration,
     init: Boolean,
+    noSecretKey: Boolean, // Default false
```

Comment:
I think you can specify the default value here, like
```suggestion
    noSecretKey: Boolean = false,
```